language/fr/tech.lang

```diff
@@ -1,5 +1,121 @@
 <?php
 
-include("{$_EnginePath}language/en/tech.lang");
+$_Lang['Tech']                  = "Technologies";
+$_Lang['Requirements']          = "Prérequis";
+$_Lang['Metal']                 = "Métal";
+$_Lang['Crystal']               = "Cristal";
+$_Lang['Deuterium']             = "Deutérium";
+$_Lang['Energy']                = "Énergie";
+$_Lang['DarkEnergy']            = "Énergie noire";
+$_Lang['Message']               = "Messages";
+$_Lang['GoToShop']              = 'Boutique';
+$_Lang['level']                 = "Niveau";
+$_Lang['treeinfo']              = "[i]";
+
+$_Lang['Metal_rec']             = 'de métal';
+$_Lang['Crystal_rec']           = 'de cristal';
+$_Lang['Deuterium_rec']         = 'de deutérium';
+
+$_Lang['PrevPlanet']            = "Planète précédente";
+$_Lang['NextPlanet']            = "Planète suivante";
+
+$_Lang['type_mission'][1]       = "Attaquer";
+$_Lang['type_mission'][2]       = "Attaque groupée";
+$_Lang['type_mission'][3]       = "Transporter";
+$_Lang['type_mission'][4]       = "Stationner";
+$_Lang['type_mission'][5]       = "Défendre";
+$_Lang['type_mission'][6]       = "Espionner";
+$_Lang['type_mission'][7]       = "Coloniser";
+$_Lang['type_mission'][8]       = "Recycler";
+$_Lang['type_mission'][9]       = "Destruire";
+$_Lang['type_mission'][10]      = 'Attaque interplanétaire';
+$_Lang['type_mission'][11]      = 'Attaque groupée';
+$_Lang['type_mission'][15]      = "Expédition";
+$_Lang['type_mission_quickres'] = 'Transport de ressources';
+
+$_Lang['tech'] = array
+(
+    0 => "Bâtiments",
+    1 => "Mine de métal",
+    2 => "Mine de cristal",
+    3 => "Synthétiseur de deutérium",
+    4 => "Centrale électrique solaire",
+    12 => "Centre électrique de fusion",
+    14 => "Usine de robots",
+    15 => "Usine de nanites",
+    21 => "Chantier spatial",
+    22 => "Hangar de métal",
+    23 => "Hangar de cristal",
+    24 => "Réservoir de deutérium",
+    31 => "Centre de recherche",
+    33 => "Terraformeur",
+    34 => "Dépôt de ravitaillement",
+
+    40 => 'Bâtiments spéciaux',
+    41 => 'Base lunaire',
+    42 => 'Phalange de capteur',
+    43 => 'Porte de saut spatial',
+    44 => "Silo de missiles",
+    50 => "Porte quantique",
+
+    100 => 'Recherche',
+    106 => 'Espionnage',
+    108 => 'Ordinateur',
+    109 => 'Armes',
+    110 => 'Bouclier',
+    111 => 'Coque',
+    113 => 'Énergie',
+    114 => 'Hyperespace',
+    115 => 'Combustion',
```

Comment:
Both `Combustion drive` and `Impulse drive` translations seem to be incomplete (online translation says it doesn't mention the "drive" part). Do you think these should be changed (eg. to something similar to the `Hyperspace drive` -> `Propulsion hyperespace`)?

Reply:
I understand your point. I added "Réacteur à" for "combustion" and "impulsion", to have the same translation as OG.